sidebars.js

```diff
@@ -79,13 +79,12 @@ module.exports = {
         type: 'category',
         label: 'Guides',
         items: [
-          'java/guides/sap-cloud-sdk-linux-how-to',
```

Comment:
I reordered a bit according to the frequency with which the information would probably be accessed.